docs/editor/variables-reference.md

```diff
@@ -83,9 +83,9 @@ You can reference VS Code settings (aka "configurations") through **${config:Nam
 
 If the predefined variables from above are not sufficient, you can use any VS Code command as a variable through the **${command:commandID}** syntax.
 
-When a command variable is interpolated, the command is run and the variable is substituted by the command's (string) result. The implementation of a command can range from a simple calculation with no UI, to some sophisticated functionality based on the UI features available via VS Code's extension API.
```

Comment:
Have to disagree. This whole sentence is incorrect and part of my confusion.  It describes the logic I implemented in my PR and not the current logic as we have in vscode.

It states that at the time of command variable alter/insert (interpolated is a strange wording choice), the command is run and that the variable is replaced by the result.

If something is needed here, it needs to be akin to this:

> A command variable is replaced with the (string) result from the command evaluation.  Evaluation of all commands occurs before any replacements (see FAQ).  The implementation ...

Reply:
I like the sentence:
> A command variable is replaced with the (string) result from the command evaluation.

Let's use that.

But I would not add the next sentence 
> Evaluation of all commands occurs before any replacements (see FAQ).

because it is confusing.
Here is why: first and foremost the variables documentation is for end users and not for extension authors.
Users don't know whether a command variable depends on other attributes in the underlying JSON. So for them the sentence "Evaluation of all commands occurs before ..." is not really helpful.